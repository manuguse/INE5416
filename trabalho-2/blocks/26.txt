1 1 8 9 9 14 16 16 16 16
1 1 8 8 10 14 16 19 16 16
2 1 1 8 10 14 14 19 19 19
2 2 2 10 10 14 14 14 19 21
2 6 6 10 10 15 17 17 21 21
2 3 6 7 12 12 12 12 21 23
3 3 6 7 12 12 18 12 21 21
3 7 7 7 7 13 18 18 22 22
4 4 5 5 13 13 13 18 22 22
5 5 5 11 11 13 13 20 20 20