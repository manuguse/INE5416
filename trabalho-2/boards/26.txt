6 - - - 2 - - - 2 4
- 3 2 4 - - - 4 - -
- 2 5 - 4 - - - - 2
5 - - - - - - 2 - -
3 1 - - - - 2 - 5 2
- - - 5 - - - 7 - -
2 - - 3 5 - 2 - - 3
- 2 - - 6 - - 4 - 4
- - 3 - 3 - - - - 3
- 4 - - - 1 5 - 3 2