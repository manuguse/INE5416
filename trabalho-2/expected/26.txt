6 4 3 1 2 7 6 7 2 4
1 3 2 4 5 6 5 4 1 3
6 2 5 1 4 5 3 1 5 2
5 4 2 6 3 4 1 2 3 6
3 1 4 1 2 1 2 1 5 2
1 4 3 5 6 2 4 7 4 1
2 3 2 3 5 1 2 3 1 3
1 2 4 1 6 4 1 4 2 4
2 1 3 5 3 2 6 3 1 3
1 4 2 1 2 1 5 1 3 2